src/odemis/gui/cont/acquisition/cryo_z_localization.py

```diff
@@ -68,51 +74,308 @@ def __init__(self, tab_data, panel, tab):
             self._stigmator.moveAbs({"rz": 0})
 
         # If the hardware doesn't support for Z localization, hide everything and don't control anything
-        if not hasattr(tab_data, "stigmatorAngle"):
+        if not hasattr(tab_data, "stigmatorAngle") and not hasattr(tab_data, "fiducial_size") and not hasattr(tab_data, "poi_size"):
```

Comment:
It should be possible to just write `if not hasattr(tab_data, "fiducial_size")`, assuming you only have the new style of calibration.